DQMOffline/EGamma/python/egammaDQMOffline_cff.py

```diff
@@ -36,7 +36,7 @@
 stdPhotonAnalysisHGCal = stdPhotonAnalysis.clone()
 stdPhotonAnalysisHGCal.ComponentName = 'stdPhotonAnalyzerHGCalFromMultiCl'
```

Comment:
Not sure if I understand if you would like to keep the ComponentName and analyzerName as before, `stdPhotonAnalyzerHGCalFromMultiCl`. 

This comment includes 
https://github.com/cms-sw/cmssw/blob/master/DQMOffline/EGamma/python/photonOfflineDQMClient_cff.py#L13

Reply:
Sorry for late comment @SohamBhattacharya 
I just look through the change in the histogram and have a question.

Reply:
@srimanob I had left some of the names in the DQM modules unchanged as the DQM updates are to follow soon [1]. However, it's true that it would be good to update the names to be consistent with the new naming convention.
After consulting our DQM/Validation experts (@beaudett  and @archiron), I have updated the names in the latest commit.

[1] https://github.com/cms-sw/cmssw/pull/34439#issuecomment-886063450

Reply:
Thanks @SohamBhattacharya

Reply:
It would be better to change this all to be
```python
stdPhotonAnalysisHGCal = stdPhotonAnalysis.clone(
  ComponentName = 'stdPhotonAnalyzerHGCalFromMultiCl',
  analyzerName = 'stdPhotonAnalyzerHGCalFromMultiCl',
  phoProducer = 'photonsHGC',
  isolationStrength = 2,
  etaMin = -3.0,
  etaMax = 3.0,
  maxPhoEta = 3.0
)
```